### PR TITLE
Add documentation for Lightwave lights and switches

### PR DESCRIPTION
**Pull request by GeoffAtHome**
**Description:**

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#<home-assistant PR number goes here>

## Checklist:

- [ ] Branch: `next` is for changes and new documentation that will go public with the next [home-assistant](https://github.com/home-assistant/home-assistant) release. Fixes, changes and adjustments for the current release should be created against `current`.
- [ ] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

File: source/_components/light.lightwave.markdown [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+title: "Lightwave Light"
+description: "Instructions on how to integrate Lightwave lights with Home Assistant."
+date: 2018-12-02 18:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: lightwave.png
+ha_category: Light
+ha_release: 0.1
```
Comment (MartinHjelmare) [inline source/_components/light.lightwave.markdown]: 0.84

File: source/_components/light.lightwave.markdown [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+ha_iot_class: "Local Polling"
```
Comment (MartinHjelmare) [inline source/_components/light.lightwave.markdown]: Assumed State

File: source/_components/light.lightwave.markdown [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+---
+The `lightwave` light platform integrates your Lightwave lights into Home Assistant.
+
+The first use of a light will try to register with the WiFi Link hub. If the hub has not been registered a message on your WiFi Link asking you to pair the device. You have 12 seconds to push the button on the WiFi Link to accept this. Once done, you should be able to control your lights via Home Asssistant. This only needs to be done if the hub has not been registered.
```
Comment (MartinHjelmare) [inline source/_components/light.lightwave.markdown]: The second sentence is missing a verb for the message.

File: source/_components/lightwave.markdown [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+title: "Lightwave"
+description: "Instructions on how to integrate Lightwave devices with Home Assistant."
+date: 2018-12-02 18:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: lightwave.png
+ha_category: Hub
+ha_release: 0.1
```
Comment (MartinHjelmare) [inline source/_components/lightwave.markdown]: 0.84

File: source/_components/lightwave.markdown [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+ha_iot_class: "Local Polling"
```
Comment (MartinHjelmare) [inline source/_components/lightwave.markdown]: See above.

File: source/_components/lightwave.markdown [hunk 6]
```diff
@@ -0,0 +1,42 @@
...
+---
+The `lightwave` component is the main component to set up and supported Lightwave lights and switches. Lights and switches need to be added in their own sections
```
Comment (MartinHjelmare) [inline source/_components/lightwave.markdown]: ... to set up the supported Lightwave lights and switches.

Comment (MartinHjelmare) [reply source/_components/lightwave.markdown]: Missing period at the end.

Comment (MartinHjelmare) [reply source/_components/lightwave.markdown]: It's a bit unclear what we mean with "need to be added in their own sections".

File: source/_components/lightwave.markdown [hunk 7]
```diff
@@ -0,0 +1,42 @@
...
+      name: Radio socket
+    R2D2:
+      name: Light socket
+    R2D3:
+      name: Phone socket
+    R2D4:
+      name: Torch socket
+```
+Where *192.168.1.2* is the ip address of your Lightwave hub.
+The Lightwave Home Assistant platform currently supports the following Lightwave devices:
```
Comment (MartinHjelmare) [inline source/_components/lightwave.markdown]: This is the component not the platform.

File: source/_components/switch.lightwave.markdown [hunk 8]
```diff
@@ -0,0 +1,19 @@
...
+title: "Lightwave Switch"
+description: "Instructions on how to integrate Lightwave Switches with Home Assistant."
+date: 2018-12-02 18:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: lightwave.png
+ha_category: Switch
+ha_release: 0.1
```
Comment (MartinHjelmare) [inline source/_components/switch.lightwave.markdown]: 0.84

File: source/_components/switch.lightwave.markdown [hunk 9]
```diff
@@ -0,0 +1,19 @@
...
+ha_iot_class: "Local Polling"
```
Comment (MartinHjelmare) [inline source/_components/switch.lightwave.markdown]: See above.

File: source/_components/switch.lightwave.markdown [hunk 10]
```diff
@@ -0,0 +1,19 @@
...
+---
+The `lightwave` switch platform integrates your Lightwave switches into Home Assistant.
+The first use of a switch will try to register with the WiFi Link hub. If the hub has not been registered a message on your WiFi Link asking you to pair the device. You have 12 seconds to push the button on the WiFi Link to accept this. Once done, you should be able to control your switches via Home Asssistant. This only needs to be done if the hub has not been registered.
```
Comment (MartinHjelmare) [inline source/_components/switch.lightwave.markdown]: The second sentence is missing a verb.

**Comment by GeoffAtHome** [comment]
@MartinHjelmare thanks for the detailed review. The terms used are now making more sense to me. I'll updated and check-in this evening.

File: source/_components/light.lightwave.markdown (hunk 1 above)
Comment (GeoffAtHome) [reply source/_components/light.lightwave.markdown]: Done

File: source/_components/lightwave.markdown (hunk 4 above)
Comment (GeoffAtHome) [reply source/_components/lightwave.markdown]: Done

File: source/_components/lightwave.markdown (hunk 5 above)
Comment (GeoffAtHome) [reply source/_components/lightwave.markdown]: Done.

File: source/_components/light.lightwave.markdown (hunk 2 above)
Comment (GeoffAtHome) [reply source/_components/light.lightwave.markdown]: Done.

File: source/_components/switch.lightwave.markdown (hunk 8 above)
Comment (GeoffAtHome) [reply source/_components/switch.lightwave.markdown]: Done.

File: source/_components/switch.lightwave.markdown (hunk 9 above)
Comment (GeoffAtHome) [reply source/_components/switch.lightwave.markdown]: Done.

File: source/_components/lightwave.markdown (hunk 7 above)
Comment (GeoffAtHome) [reply source/_components/lightwave.markdown]: Done.

File: source/_components/light.lightwave.markdown (hunk 3 above)
Comment (GeoffAtHome) [reply source/_components/light.lightwave.markdown]: Text updated.

File: source/_components/lightwave.markdown (hunk 6 above)
Comment (GeoffAtHome) [reply source/_components/lightwave.markdown]: Revised the text.

File: source/_components/switch.lightwave.markdown (hunk 10 above)
Comment (GeoffAtHome) [reply source/_components/switch.lightwave.markdown]: Revised the text.

File: source/_components/lightwave.markdown [hunk 11]
```diff
@@ -0,0 +1,49 @@
...
+      name: Radio socket
+    R2D2:
+      name: Light socket
+    R2D3:
+      name: Phone socket
+    R2D4:
+      name: Torch socket
+```
+
+Where ```192.168.1.2``` is the ip address of your Lightwave hub.
```
Comment (MartinHjelmare) [inline source/_components/lightwave.markdown]: Single backtics on either side of the word is enough.

Comment (GeoffAtHome) [reply source/_components/lightwave.markdown]: Done and learnt something new about markdown.

File: source/_components/light.lightwave.markdown [hunk 12]
```diff
@@ -0,0 +1,20 @@
...
+sharing: true
+footer: true
+logo: lightwave.png
+ha_category: Light
+ha_release: 0.84
+ha_iot_class: "Assumed State"
+---
+The `lightwave` light platform integrates your Lightwave lights into Home Assistant.
+
+The first use of a light will try to register with your Lightwave WiFi Link hub. If the hub has not been registered a message on your hub will be displayed asking you to pair the device. You have 12 seconds to push the button on your hub to accept this. Once done, you should be able to control your lights via Home Assistant. This only needs to be done if the hub has not been registered.
```
Comment (MartinHjelmare) [inline source/_components/light.lightwave.markdown]: Maybe it's enough to only have this on the hub/component page? You decide.

Comment (GeoffAtHome) [reply source/_components/light.lightwave.markdown]: Agree with KISS. Less is more. I have removed the unnecessary files and update the code that referenced these files.

**Comment by MartinHjelmare** [comment]
We still want to have those pages for the platforms. I was only referring to the paragraph that was copied to all three pages. We can keep that paragraph in the component page.

**Comment by GeoffAtHome** [comment]
@MartinHjelmare Revert the file deletes and removed repeated paragraph.

**Review by MartinHjelmare — APPROVED** [review]

**Pull request merged**